src/output-json-drop.c

```diff
@@ -60,7 +61,8 @@
 
 #define MODULE_NAME "JsonDropLog"
 
-#define LOG_DROP_ALERTS 1
+#define LOG_DROP_ALERTS  BIT_U8(1)
+#define LOG_DROP_VERDICT BIT_U8(2)
```

Comment:
Changed this, tried adding the flows start/all option here, but led to failures. Can make this a TODO.